docs/source/guides/links.rst

```diff
@@ -0,0 +1,89 @@
+Links
+~~~~~
+
+In order to write neural networks, we have to combine functions with *parameters* and optimize the parameters.
+You can use **links** to do this.
+A link is an object that holds parameters (i.e. optimization targets).
+
+The most fundamental ones are links that behave like regular functions while replacing some arguments by their parameters.
+We will introduce higher level links, but here think of links as simply functions with parameters.
+
+One of the most frequently used links is the :class:`~functions.Linear` link (a.k.a. *fully-connected layer* or *affine transformation*).
+It represents a mathematical function :math:`f(x) = Wx + b`, where the matrix :math:`W` and the vector :math:`b` are parameters.
+This link corresponds to its pure counterpart :func:`~functions.linear`, which accepts :math:`x, W, b` as arguments.
+A linear link from three-dimensional space to two-dimensional space is defined by the following line:
+
+.. doctest::
+
+   >>> f = L.Linear(3, 2)
+
+.. note::
+   Most functions and links only accept mini-batch input, where the first dimension of the input array is considered as the *batch dimension*.
+   In the above Linear link case, input must have shape of (N, 3), where N is the mini-batch size.
+
+The parameters of a link are stored as attributes.
+Each parameter is an instance of :class:`~chainer.Variable`.
+In the case of the Linear link, two parameters, ``W`` and ``b``, are stored.
+By default, the matrix ``W`` is initialized randomly, while the vector ``b`` is initialized with zeros.
```

Comment:
How about adding a line to explain that is the generally preferred way to initialize these parameters.

Reply:
Added `This is the preferred way to initialize these parameters.`